ext/standard/string.c

```diff
@@ -576,7 +576,7 @@ static zend_always_inline zend_string *php_trim_int(zend_string *str, const char
 				unsigned char c = (unsigned char)*start;
 
 				if (c <= ' ' &&
-				    (c == ' ' || c == '\n' || c == '\r' || c == '\t' || c == '\v' || c == '\0')) {
+				    (c == ' ' || c == '\f' || c == '\n' || c == '\r' || c == '\t' || c == '\v' || c == '\0')) {
```

Comment:
nit: personal opinion would be nice to share a helper for both cases e.g.

```c
static zend_always_inline bool php_is_whitespace(char c)
{
    ...
}
```

but no need to bother now until the topic is discussed at least.